css/css-pseudo/marker-variable.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="author" title="Karl Dubost" href="https://github.com/karlcow" />
+    <link
+      rel="help"
+      href="https://w3c.github.io/csswg-drafts/css-pseudo/#marker-pseudo"
+    />
+    <link
+      rel="help"
+      href="https://w3c.github.io/csswg-drafts/css-variables/#defining-variables"
+    />
```

Comment:
This needs a `<link rel="match">` to your `-ref.html` file. Otherwise this won't get detected as a reftest.

Reply:
@nt1m I added a rel match, but also added an automated test.

One thing is puzzling me. I added an automated test. 
Firefox does the right thing but fails the test, while Chrome is passing. 
@jgraham

Reply:
@karlcow I think Firefox behaves correctly, `getComputedStyle(firstTest, 'marker');` somehow works in Chromium, when it should be `getComputedStyle(firstTest, '::marker');`

Reply:
I wonder later if there is a test about invalid pseudo-selector needed. but that would be for a separate pr.